learn_django/trip_project/urls.py

```diff
@@ -9,5 +9,6 @@
 
 urlpatterns = [
     url('api/v1/', include(router.urls)),
-    url(r'^trips$', views.index, name='index')
```

Comment:
why you have removed this?

Reply:
because I though it was of no use, as we have api/v1/trips and api/v1/trip so keeping /trips is of no use